### PR TITLE
Understanding Buffer Chunk Limiting for Fluentd documentation is missing the prerequisite section

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1827182

### CODE REVIEW

**Comment by mburke5678** [comment]
@periklis @koep Please see my questions in https://bugzilla.redhat.com/show_bug.cgi?id=1827182#c1

**Comment by mburke5678** [comment]
@periklis @koep   PTAL

**Comment by koep** [comment]
@mburke5678 LGTM

File: modules/nodes-cluster-overcommit-buffer-chunk.adoc [hunk 1]
```diff
@@ -12,6 +12,12 @@ to switch to file buffering to reduce memory usage and prevent data loss.
 
 Fluentd file buffering stores records in _chunks_. Chunks are stored in _buffers_.
 
+[NOTE]
+====
+To modify the `BUFFER_QUEUE_LIMIT`, `FILE_BUFFER_LIMIT`, or `BUFFER_SIZE_LIMIT` parameters
```
Comment (periklis) [inline modules/nodes-cluster-overcommit-buffer-chunk.adoc:17]: As mentioned in [BZ-1827182](https://bugzilla.redhat.com/show_bug.cgi?id=1827182) you a cannot modify `BUFFER_QUEUE_LIMIT` directly.

**Review by periklis — CHANGES_REQUESTED** [review]

**Review by periklis — APPROVED** [review]
LGTM

**Comment by xltian** [comment]
@anpingli Can you please take a look at this change?

**Review by anpingli — APPROVED** [review]
LGTM

**Pull request merged**